test/parallel/test-fs-promises-writefile.js

```diff
@@ -25,9 +29,33 @@ async function doWriteWithCancel() {
   const controller = new AbortController();
   const { signal } = controller;
   process.nextTick(() => controller.abort());
-  assert.rejects(fsPromises.writeFile(otherDest, buffer, { signal }), {
+  assert.rejects(fsPromises.writeFile(otherDest2, buffer, { signal }), {
     name: 'AbortError'
-  });
+  }).then(common.mustCall(()=> {
```

Comment:
```suggestion
  }).then(common.mustCall(() => {
```
To fix the linter errors.